fix: heading examples (#1687)
contentful/forma-36: packages/components/typography/src/Heading.mdx

section: 'typographyComponents'
slug: /components/heading/
github: 'https://github.com/contentful/forma-36/tree/forma-v4/packages/components/typography/src'
typescript: ./Heading.tsx
---

> `Heading` is intended to be used to define a title of the page. They are displayed as `h1`. Accessibility requires to have just one `h1` tag on the page. Keep that in mind during the implementation.

Reviewer: ```suggestion
`Heading` is intended to be used to define the title of the page. They are displayed as `h1` by default. Accessibility requires having just one `h1` tag on the page. Keep that in mind during the implementation.
```

## Import

```js static=true
import { Heading } from '@contentful/f36-components';
```

## Examples

```jsx file=examples/HeadingExample.tsx

```

### Margins

> Forma 36 typography components have a possiblity to set their own margins. By default `Heading` component has a margin bottom of `1rem` (`16px`). You can override the default margin bottom by specifying it explicitly on the component level by `marginBottom="none"` or any other value from [our spacing options](/tokens/spacing)

Reviewer 2: ```suggestion
Forma 36 typography components provide a way to set their own margins. By default `Heading` component has a margin bottom of `1rem` (`16px`). You can override the default margin bottom by specifying it explicitly on the component level by `marginBottom="none"` or any other value from [our spacing options](/tokens/spacing)
```
